src/components/NewOrder.js

```diff
@@ -46,6 +46,12 @@ class NewOrder extends Component {
     this.setState({ comment });
   }
 
+  removeEntry(i) {
+    const updatedEntries = [...this.state.entries.splice(0, i), ...this.state.entries.splice(i + 1)];
```

Comment:
Do you mean `slice`? I think `splice` is destructive.

We might also be able to just pass the `entry`, and do something like `const updatedEntries = _.without(this.state.entries, entry);`.